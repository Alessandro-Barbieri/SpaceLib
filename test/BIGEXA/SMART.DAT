6

0
0  0.8  0  0  1.57079
600
100 0   0
  120   0
      100
0 -0.40 0


0
0  0  0  0.75  0
300
18 0  0
   10 0
     10
-0.30 0 0

0
0  0  0.5  0  -1.57079
200
10   0 0
  10   0
      18
0 0 -0.2

0
0  0.45  0  0  1.57079
100
3 0 0
  5 0
    3
0 -0.20 0

0
0  0  0.15  0  -1.57079
100
3 0 0
  3 0
    5
0 0 -0.07

0
0  0.0  0  0  0
200
4 0 0
  4 0
    6
0 0 -0.5

0 0 -9.8
0 0 0 0 0 0
