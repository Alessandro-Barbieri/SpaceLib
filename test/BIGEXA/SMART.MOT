0.02
0	0.5	2
1.57	0.5	2
-1.57	0.5	2
0	0.5	2
0.26	0.5	2
0	0.5	2	