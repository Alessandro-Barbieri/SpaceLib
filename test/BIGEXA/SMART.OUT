

 Link 1


 Relative Position Matrix                   A

  1.0000	  0.0000	  0.0000	  0.0000
  0.0000	  0.0000	 -1.0000	  0.0000
  0.0000	  1.0000	  0.0000	  0.8000
  0.0000	  0.0000	  0.0000	  1.0000

 Absolute Position Matrix                   T

  1.0000	  0.0000	  0.0000	  0.0000
  0.0000	  0.0000	 -1.0000	  0.0000
  0.0000	  1.0000	  0.0000	  0.8000
  0.0000	  0.0000	  0.0000	  1.0000

 Relative Velocity Matrix in frame (i)      W

  0.0000	 -0.5000	  0.0000	  0.0000
  0.5000	  0.0000	  0.0000	  0.0000
  0.0000	  0.0000	  0.0000	  0.0000
  0.0000	  0.0000	  0.0000	  0.0000

 Relative Velocity Matrix in frame (0)     WO

  0.0000	 -0.5000	  0.0000	  0.0000
  0.5000	  0.0000	  0.0000	  0.0000
  0.0000	  0.0000	  0.0000	  0.0000
  0.0000	  0.0000	  0.0000	  0.0000

 Absolute Velocity Matrix in frame (0)     WA

  0.0000	 -0.5000	  0.0000	  0.0000
  0.5000	  0.0000	  0.0000	  0.0000
  0.0000	  0.0000	  0.0000	  0.0000
  0.0000	  0.0000	  0.0000	  0.0000

 Relative Acceleration Matrix in frame (i)  H

 -0.2500	 -2.0000	  0.0000	  0.0000
  2.0000	 -0.2500	  0.0000	  0.0000
  0.0000	  0.0000	  0.0000	  0.0000
  0.0000	  0.0000	  0.0000	  0.0000

 Relative Acceleration Matrix in frame  (0)HO

 -0.2500	 -2.0000	  0.0000	  0.0000
  2.0000	 -0.2500	  0.0000	  0.0000
  0.0000	  0.0000	  0.0000	  0.0000
  0.0000	  0.0000	  0.0000	  0.0000

 Absolute Acceleration Matrix in frame  (0) A

 -0.2500	 -2.0000	  0.0000	  0.0000
  2.0000	 -0.2500	  0.0000	  0.0000
  0.0000	  0.0000	  0.0000	  0.0000
  0.0000	  0.0000	  0.0000	  0.0000

 Inertia Matrix in frame (i)                J

 60.0000	  0.0000	  0.0000	  0.0000
  0.0000	 136.0000	  0.0000	 -240.0000
  0.0000	  0.0000	 60.0000	  0.0000
  0.0000	 -240.0000	  0.0000	 600.0000

 Inertia Matrix in frame (0)                O

 60.0000	  0.0000	  0.0000	  0.0000
  0.0000	 60.0000	 -0.0007	 -0.0015
  0.0000	 -0.0007	 136.0000	 240.0000
  0.0000	 -0.0015	 240.0000	 600.0000

 Total actions                             FI

  0.0000	 240.0000	 -0.0015	 -0.0030
 -240.0000	  0.0000	 -0.0151	 -0.0004
  0.0015	  0.0151	  0.0000	 -5880.0000
  0.0030	  0.0004	 5880.0000	  0.0000

 Actions on Joint (i)                    ACTO

  0.0000	 381.1567	 5663.0017	 2542.5464
 -381.1567	  0.0000	 1104.5747	 527.9832
 -5663.0017	 -1104.5747	  0.0000	 -14365.1269
 -2542.5464	 -527.9832	 14365.1269	  0.0000


 Link 2


 Relative Position Matrix                   A

  0.0008	 -1.0000	  0.0000	  0.0006
  1.0000	  0.0008	  0.0000	  0.7500
  0.0000	  0.0000	  1.0000	  0.0000
  0.0000	  0.0000	  0.0000	  1.0000

 Absolute Position Matrix                   T

  0.0008	 -1.0000	  0.0000	  0.0006
  0.0000	  0.0000	 -1.0000	  0.0000
  1.0000	  0.0008	  0.0000	  1.5500
  0.0000	  0.0000	  0.0000	  1.0000

 Relative Velocity Matrix in frame (i)      W

  0.0000	 -0.5000	  0.0000	  0.0000
  0.5000	  0.0000	  0.0000	  0.0000
  0.0000	  0.0000	  0.0000	  0.0000
  0.0000	  0.0000	  0.0000	  0.0000

 Relative Velocity Matrix in frame (0)     WO

  0.0000	 -0.0000	 -0.5000	  0.4000
  0.0000	  0.0000	  0.0000	  0.0000
  0.5000	  0.0000	  0.0000	  0.0000
  0.0000	  0.0000	  0.0000	  0.0000

 Absolute Velocity Matrix in frame (0)     WA

  0.0000	 -0.5000	 -0.5000	  0.4000
  0.5000	  0.0000	  0.0000	  0.0000
  0.5000	  0.0000	  0.0000	  0.0000
  0.0000	  0.0000	  0.0000	  0.0000

 Relative Acceleration Matrix in frame (i)  H

 -0.2500	 -2.0000	  0.0000	  0.0000
  2.0000	 -0.2500	  0.0000	  0.0000
  0.0000	  0.0000	  0.0000	  0.0000
  0.0000	  0.0000	  0.0000	  0.0000

 Relative Acceleration Matrix in frame  (0)HO

 -0.2500	 -0.0000	 -2.0000	  1.6000
  0.0000	 -0.0000	 -0.0000	  0.0000
  2.0000	 -0.0000	 -0.2500	  0.2000
  0.0000	  0.0000	  0.0000	  0.0000

 Absolute Acceleration Matrix in frame  (0) A

 -0.5000	 -2.0000	 -2.0000	  1.6000
  2.0000	 -0.2500	 -0.5000	  0.4000
  2.0000	 -0.0000	 -0.2500	  0.2000
  0.0000	  0.0000	  0.0000	  0.0000

 Inertia Matrix in frame (i)                J

 28.0000	  0.0000	  0.0000	 -90.0000
  0.0000	  9.0000	  0.0000	  0.0000
  0.0000	  0.0000	  9.0000	  0.0000
 -90.0000	  0.0000	  0.0000	 300.0000

 Inertia Matrix in frame (0)                O

  9.0000	  0.0000	  0.1280	  0.1075
  0.0000	  9.0000	  0.0010	  0.0009
  0.1280	  0.0010	 469.7499	 375.0000
  0.1075	  0.0009	 375.0000	 300.0000

 Total actions                             FI

  0.0000	 35.9800	 358.6090	 270.0554
 -35.9800	  0.0000	 84.6277	 67.2854
 -358.6090	 -84.6277	  0.0000	 -2906.4650
 -270.0554	 -67.2854	 2906.4650	  0.0000

 Actions on Joint (i)                    ACTO

  0.0000	 141.1567	 5663.0032	 2542.5494
 -141.1567	  0.0000	 1104.5898	 527.9836
 -5663.0032	 -1104.5898	  0.0000	 -8485.1269
 -2542.5494	 -527.9836	 8485.1269	  0.0000


 Link 3


 Relative Position Matrix                   A

  0.0008	  0.0000	  1.0000	  0.5000
 -1.0000	  0.0000	  0.0008	  0.0004
  0.0000	 -1.0000	  0.0000	  0.0000
  0.0000	  0.0000	  0.0000	  1.0000

 Absolute Position Matrix                   T

  1.0000	  0.0000	  0.0000	  0.0006
  0.0000	  1.0000	  0.0000	  0.0000
  0.0000	  0.0000	  1.0000	  2.0500
  0.0000	  0.0000	  0.0000	  1.0000

 Relative Velocity Matrix in frame (i)      W

  0.0000	 -0.5000	  0.0000	  0.0000
  0.5000	  0.0000	  0.0000	  0.0000
  0.0000	  0.0000	  0.0000	  0.0000
  0.0000	  0.0000	  0.0000	  0.0000

 Relative Velocity Matrix in frame (0)     WO

  0.0000	 -0.0000	 -0.5000	  0.7750
  0.0000	  0.0000	  0.0000	 -0.0000
  0.5000	  0.0000	  0.0000	 -0.0003
  0.0000	  0.0000	  0.0000	  0.0000

 Absolute Velocity Matrix in frame (0)     WA

  0.0000	 -0.5000	 -1.0000	  1.1750
  0.5000	  0.0000	  0.0000	 -0.0000
  1.0000	  0.0000	  0.0000	 -0.0003
  0.0000	  0.0000	  0.0000	  0.0000

 Relative Acceleration Matrix in frame (i)  H

 -0.2500	 -2.0000	  0.0000	  0.0000
  2.0000	 -0.2500	  0.0000	  0.0000
  0.0000	  0.0000	  0.0000	  0.0000
  0.0000	  0.0000	  0.0000	  0.0000

 Relative Acceleration Matrix in frame  (0)HO

 -0.2500	 -0.0000	 -2.0000	  3.1001
  0.0000	 -0.0000	 -0.0000	  0.0000
  2.0000	 -0.0000	 -0.2500	  0.3863
  0.0000	  0.0000	  0.0000	  0.0000

 Absolute Acceleration Matrix in frame  (0) A

 -1.2500	 -2.0000	 -4.0000	  4.7004
  2.0000	 -0.2500	 -1.0000	  1.1750
  4.0000	 -0.0000	 -1.0000	  1.3613
  0.0000	  0.0000	  0.0000	  0.0000

 Inertia Matrix in frame (i)                J

  9.0000	  0.0000	  0.0000	  0.0000
  0.0000	  9.0000	  0.0000	  0.0000
  0.0000	  0.0000	  9.0000	 -40.0000
  0.0000	  0.0000	 -40.0000	 200.0000

 Inertia Matrix in frame (0)                O

  9.0001	  0.0000	  0.2210	  0.1194
  0.0000	  9.0000	  0.0029	  0.0016
  0.2210	  0.0029	 685.4998	 370.0000
  0.1194	  0.0016	 370.0000	 200.0000

 Total actions                             FI

  0.0000	 35.9242	 1040.2286	 540.0628
 -35.9242	  0.0000	 250.3245	 134.7621
 -1040.2286	 -250.3245	  0.0000	 -1862.7389
 -540.0628	 -134.7621	 1862.7389	  0.0000

 Actions on Joint (i)                    ACTO

  0.0000	 105.1768	 5304.3942	 2272.4940
 -105.1768	  0.0000	 1019.9620	 460.6982
 -5304.3942	 -1019.9620	  0.0000	 -5578.6619
 -2272.4940	 -460.6982	 5578.6619	  0.0000


 Link 4


 Relative Position Matrix                   A

  1.0000	  0.0000	  0.0000	  0.0000
  0.0000	  0.0000	 -1.0000	  0.0000
  0.0000	  1.0000	  0.0000	  0.4500
  0.0000	  0.0000	  0.0000	  1.0000

 Absolute Position Matrix                   T

  1.0000	  0.0000	  0.0000	  0.0006
  0.0000	  0.0000	 -1.0000	  0.0000
  0.0000	  1.0000	  0.0000	  2.5000
  0.0000	  0.0000	  0.0000	  1.0000

 Relative Velocity Matrix in frame (i)      W

  0.0000	 -0.5000	  0.0000	  0.0000
  0.5000	  0.0000	  0.0000	  0.0000
  0.0000	  0.0000	  0.0000	  0.0000
  0.0000	  0.0000	  0.0000	  0.0000

 Relative Velocity Matrix in frame (0)     WO

  0.0000	 -0.5000	  0.0000	  0.0000
  0.5000	  0.0000	  0.0000	 -0.0003
  0.0000	  0.0000	  0.0000	  0.0000
  0.0000	  0.0000	  0.0000	  0.0000

 Absolute Velocity Matrix in frame (0)     WA

  0.0000	 -1.0000	 -1.0000	  1.1750
  1.0000	  0.0000	  0.0000	 -0.0003
  1.0000	  0.0000	  0.0000	 -0.0003
  0.0000	  0.0000	  0.0000	  0.0000

 Relative Acceleration Matrix in frame (i)  H

 -0.2500	 -2.0000	  0.0000	  0.0000
  2.0000	 -0.2500	  0.0000	  0.0000
  0.0000	  0.0000	  0.0000	  0.0000
  0.0000	  0.0000	  0.0000	  0.0000

 Relative Acceleration Matrix in frame  (0)HO

 -0.2500	 -2.0000	  0.0000	  0.0002
  2.0000	 -0.2500	  0.0000	 -0.0012
  0.0000	  0.0000	  0.0000	  0.0000
  0.0000	  0.0000	  0.0000	  0.0000

 Absolute Acceleration Matrix in frame  (0) A

 -2.0000	 -4.0000	 -4.0000	  4.7009
  4.0000	 -1.0000	 -1.0000	  1.1738
  4.0000	 -1.0000	 -1.0000	  1.3613
  0.0000	  0.0000	  0.0000	  0.0000

 Inertia Matrix in frame (i)                J

  2.5000	  0.0000	  0.0000	  0.0000
  0.0000	  4.5000	  0.0000	 -20.0000
  0.0000	  0.0000	  2.5000	  0.0000
  0.0000	 -20.0000	  0.0000	 100.0000

 Inertia Matrix in frame (0)                O

  2.5000	  0.0000	  0.1374	  0.0597
  0.0000	  2.5000	  0.0015	  0.0007
  0.1374	  0.0015	 529.4999	 230.0000
  0.0597	  0.0007	 230.0000	 100.0000

 Total actions                             FI

  0.0000	 19.9359	 1047.6003	 450.0309
 -19.9359	  0.0000	 256.4826	 112.3812
 -1047.6003	 -256.4826	  0.0000	 -886.3696
 -450.0309	 -112.3812	 886.3696	  0.0000

 Actions on Joint (i)                    ACTO

  0.0000	 69.2525	 4264.1656	 1732.4312
 -69.2525	  0.0000	 769.6375	 325.9361
 -4264.1656	 -769.6375	  0.0000	 -3715.9230
 -1732.4312	 -325.9361	 3715.9230	  0.0000


 Link 5


 Relative Position Matrix                   A

  0.9664	 -0.0000	 -0.2571	 -0.0386
  0.2571	  0.0000	  0.9664	  0.1450
  0.0000	 -1.0000	  0.0000	  0.0000
  0.0000	  0.0000	  0.0000	  1.0000

 Absolute Position Matrix                   T

  0.9664	 -0.0000	 -0.2571	 -0.0380
  0.0000	  1.0000	 -0.0000	  0.0000
  0.2571	 -0.0000	  0.9664	  2.6450
  0.0000	  0.0000	  0.0000	  1.0000

 Relative Velocity Matrix in frame (i)      W

  0.0000	 -0.5000	  0.0000	  0.0000
  0.5000	  0.0000	  0.0000	  0.0000
  0.0000	  0.0000	  0.0000	  0.0000
  0.0000	  0.0000	  0.0000	  0.0000

 Relative Velocity Matrix in frame (0)     WO

  0.0000	 -0.0000	 -0.5000	  1.2500
  0.0000	  0.0000	  0.0000	 -0.0000
  0.5000	  0.0000	  0.0000	 -0.0003
  0.0000	  0.0000	  0.0000	  0.0000

 Absolute Velocity Matrix in frame (0)     WA

  0.0000	 -1.0000	 -1.5000	  2.4250
  1.0000	  0.0000	  0.0000	 -0.0003
  1.5000	  0.0000	  0.0000	 -0.0006
  0.0000	  0.0000	  0.0000	  0.0000

 Relative Acceleration Matrix in frame (i)  H

 -0.2500	 -2.0000	  0.0000	  0.0000
  2.0000	 -0.2500	  0.0000	  0.0000
  0.0000	  0.0000	  0.0000	  0.0000
  0.0000	  0.0000	  0.0000	  0.0000

 Relative Acceleration Matrix in frame  (0)HO

 -0.2500	 -0.0000	 -2.0000	  5.0001
  0.0000	 -0.0000	 -0.0000	  0.0000
  2.0000	 -0.0000	 -0.2500	  0.6238
  0.0000	  0.0000	  0.0000	  0.0000

 Absolute Acceleration Matrix in frame  (0) A

 -3.2500	 -4.0000	 -6.0000	  9.7017
  4.0000	 -1.0000	 -2.0000	  3.6738
  6.0000	 -1.0000	 -2.2500	  4.4851
  0.0000	  0.0000	  0.0000	  0.0000

 Inertia Matrix in frame (i)                J

  2.5000	  0.0000	  0.0000	  0.0000
  0.0000	  2.5000	  0.0000	  0.0000
  0.0000	  0.0000	  0.9900	 -7.0000
  0.0000	  0.0000	 -7.0000	 100.0000

 Inertia Matrix in frame (0)                O

  2.4077	 -0.0000	 -4.6498	 -1.9969
 -0.0000	  2.5000	  0.0023	  0.0009
 -4.6498	  0.0023	 664.8854	 257.7311
 -1.9969	  0.0009	 257.7311	 100.0000

 Total actions                             FI

  0.0000	 21.5993	 1470.1726	 569.7344
 -21.5993	  0.0000	 399.0265	 156.0706
 -1470.1726	 -399.0265	  0.0000	 -836.6345
 -569.7344	 -156.0706	 836.6345	  0.0000

 Actions on Joint (i)                    ACTO

  0.0000	 49.3166	 3216.5654	 1282.4003
 -49.3166	  0.0000	 513.1549	 213.5549
 -3216.5654	 -513.1549	  0.0000	 -2829.5534
 -1282.4003	 -213.5549	 2829.5534	  0.0000


 Link 6


 Relative Position Matrix                   A

  1.0000	  0.0000	  0.0000	  0.0000
  0.0000	  1.0000	  0.0000	  0.0000
  0.0000	  0.0000	  1.0000	  0.0000
  0.0000	  0.0000	  0.0000	  1.0000

 Absolute Position Matrix                   T

  0.9664	 -0.0000	 -0.2571	 -0.0380
  0.0000	  1.0000	 -0.0000	  0.0000
  0.2571	 -0.0000	  0.9664	  2.6450
  0.0000	  0.0000	  0.0000	  1.0000

 Relative Velocity Matrix in frame (i)      W

  0.0000	 -0.5000	  0.0000	  0.0000
  0.5000	  0.0000	  0.0000	  0.0000
  0.0000	  0.0000	  0.0000	  0.0000
  0.0000	  0.0000	  0.0000	  0.0000

 Relative Velocity Matrix in frame (0)     WO

  0.0000	 -0.4832	 -0.0000	  0.0000
  0.4832	  0.0000	  0.1285	 -0.3216
  0.0000	 -0.1285	  0.0000	  0.0000
  0.0000	  0.0000	  0.0000	  0.0000

 Absolute Velocity Matrix in frame (0)     WA

  0.0000	 -1.4832	 -1.5000	  2.4250
  1.4832	  0.0000	  0.1285	 -0.3219
  1.5000	 -0.1285	  0.0000	 -0.0006
  0.0000	  0.0000	  0.0000	  0.0000

 Relative Acceleration Matrix in frame (i)  H

 -0.2500	 -2.0000	  0.0000	  0.0000
  2.0000	 -0.2500	  0.0000	  0.0000
  0.0000	  0.0000	  0.0000	  0.0000
  0.0000	  0.0000	  0.0000	  0.0000

 Relative Acceleration Matrix in frame  (0)HO

 -0.2335	 -1.9328	 -0.0621	  0.1554
  1.9328	 -0.2500	  0.5142	 -1.2866
 -0.0621	 -0.5142	 -0.0165	  0.0413
  0.0000	  0.0000	  0.0000	  0.0000

 Absolute Acceleration Matrix in frame  (0) A

 -4.4499	 -5.5472	 -6.3192	 10.5004
  5.9328	 -2.2164	 -1.4859	  2.3873
  5.9379	 -2.9638	 -2.2665	  4.5265
  0.0000	  0.0000	  0.0000	  0.0000

 Inertia Matrix in frame (i)                J

  3.0000	  0.0000	  0.0000	  0.0000
  0.0000	  3.0000	  0.0000	  0.0000
  0.0000	  0.0000	 51.0000	 -100.0000
  0.0000	  0.0000	 -100.0000	 200.0000

 Inertia Matrix in frame (0)                O

  4.5086	  0.0002	 39.6574	 18.1151
  0.0002	  3.0000	  0.0039	  0.0018
 39.6574	  0.0039	 935.7763	 432.3527
 18.1151	  0.0018	 432.3527	 200.0000

 Total actions                             FI

  0.0000	 27.7172	 1746.3928	 712.6660
 -27.7172	  0.0000	 114.1284	 57.4843
 -1746.3928	 -114.1284	  0.0000	 -1992.9190
 -712.6660	 -57.4843	 1992.9190	  0.0000

 Actions on Joint (i)                    ACTO

  0.0000	 27.7172	 1746.3928	 712.6660
 -27.7172	  0.0000	 114.1284	 57.4843
 -1746.3928	 -114.1284	  0.0000	 -1992.9190
 -712.6660	 -57.4843	 1992.9190	  0.0000